log4j-jul/src/main/java/org/apache/logging/log4j/jul/ApiLogger.java

```diff
@@ -91,7 +92,8 @@ public String getName() {
 
     @Override
     public void setLevel(final Level newLevel) throws SecurityException {
-        throw new UnsupportedOperationException("Cannot set level through log4j-api");
+        StatusLogger.getLogger().error("Cannot set JUL log level through log4j-api: " +
+                "ignoring call to Logger.setLevel(" + newLevel + ")");
```

Comment:
I would suggest a parameterized message, instead of string joining.

Reply:
@ppkarwasz Thank you for the suggestion. Done.